### PR TITLE
Add personal project to the website

### PR DESCRIPTION
**Pull request by mirouhml**
- Add information about capstone project for module 1 in data.js

### CODE REVIEW

File: src/data.js [hunk 1]
```diff
@@ -9,4 +9,14 @@ export default [
...
+  {
+    title: 'Module 1 capstone project - Creative Anime Convention website',
+    description: 'This website is for an anime convention it has two pages a home page and about page, the home page contains information about the convention (events, guests and sponsors) and the about page contains information about the ones holding the anime convention.',
+    author: 'Ammar Hamlaoui',
+    ghUsername: 'mirouhml',
+    repo: 'https://github.com/mirouhml/anime-convention-project',
+    license: 'MIT',
+    screenshot: 'https://github.com/mirouhml/anime-convention-project/raw/main/src/screenshot.png',
+    publishedOn: '2022-02-01',
+  }
```
Comment (ahangarha) [inline src/data.js]: Please include `,` here.

**Review by ahangarha — CHANGES_REQUESTED** [review]
Thank you for your PR. Just please fix the issue in the comment.

You PR also demonstrates a bug in the css. I changed the target from main to develop branch so that after merging your commits, I will fix that and push it to the main branch.

Thanks again.

File: src/data.js (hunk 1 above)
Comment (mirouhml) [reply src/data.js]: Done.

**Comment by mirouhml** [comment]
It's fixed thanks for pointing that out.

**Review by ahangarha — APPROVED** [review]
Thanks

**Pull request merged**